templates/launch.html

```diff
@@ -38,69 +49,69 @@ <h1>Launch a survey</h1>
         </span>
     </div>
 
-    <div class="field-container">
+    <div class="hidden-field-container">
         <span>
             <label for="schema-url">Schema URL</label>
             <input id="schema-url" name="schema_url" type="text" class="qa-schema_url">
         </span>
     </div>
 
-    <div class="field-container" id="schema-url-name">
+    <div class="hidden-field-container" id="schema-url-name">
         <span>
             <label for="schema-url-name-input">Schema Name</label>
             <input id="schema-url-name-input" name="schema_name" type="text" class="qa-schema_url_name" disabled>
         </span>
     </div>
 
-    <div class="field-container">
+    <div class="hidden-field-container">
         <input type="submit" onClick="return loadMetadataForSchemaUrl()" value="Load Schema" class="qa-btn-submit-dev btn" id="schema-url-btn"/>
     </div>
 
-    <p>----------</p>
+    <p class="group-title">----------</p>
 
-    <div id="survey_metadata_fields">
+    <div id="survey_metadata_fields" class="hidden-block">
     </div>
 
-    <h3>Survey Metadata</h3>
-    <div id="survey_metadata">
+    <h3 class="group-title">Survey Metadata</h3>
+    <div id="survey_metadata" class="hidden-block">
         <p>--- Metadata fields will be loaded when you select or load a schema ---</p>
     </div>
 
-    <h3>Required Data</h3>
+    <h3 class="group-title">Required Data</h3>
 
-    <div class="field-container">
+    <div class="hidden-field-container" id="ru_ref_container">
```

Comment:
In the v2 scenario, this is now duplicated in survey metadata bit.

I'm in favour of removing these fields, we don't need anything here that aren't top level. The only reason we may have wanted to manually set ru_ref etc is because it was used in the response_id generation which we no longer need as response_id is a required field.

Reply:
Removed this now